docs/src/guides/symfony/services/postgresql.md

```diff
@@ -129,52 +161,68 @@ You're ready to go locally with PostgreSQL.
    comment/remove `pdo_sqlite` php-ext and add `pdo_pgsql` from the list
 
 
-1. Commit your files and deploy
+4. Commit your files and deploy.
+
+   To commit your modified files to your Git branch and deploy those changes to your environment,
+   run the following command:
 
-   Commit your modified files into your git branch and then deploy it to your environment.
     ```bash
     git add . && git commit -m "change database engine to PostgreSQL" && symfony deploy
     ```
+
    After deployment, the PostgreSQL database engine is up and ready to use.
 
-1. Import data
+5. Import data.
```

Comment:
Just to make sure, do we want to import data again or is this a duplicate of step 3 above (l.87)?